googletest/include/gtest/gtest.h

```diff
@@ -2116,9 +2116,14 @@ GTEST_API_ AssertionResult DoubleLE(const char* expr1, const char* expr2,
 // of the dummy variable name, thus allowing multiple SCOPED_TRACE()s
 // to appear in the same block - as long as they are on different
 // lines.
-#define SCOPED_TRACE(message) \
+#define SCOPED_TRACE(...) GTEST_VA_SELECT_(SCOPED_TRACE, __VA_ARGS__) 
+#define SCOPED_TRACE_1(message) \
```

Comment:
Okay, yes, the declaration name doesn't matter a whole lot, so it could be `__LINE__` always --- but the ScopedTrace() constructor, if called with one parameter, can't derive the correct `__LINE__`, so the message output would be weird